Packs/Campaign/Playbooks/Detect_&_Manage_Phishing_Campaigns.yml

```diff
@@ -1206,13 +1560,28 @@ inputs:
   playbookInputQuery:
 - key: AutomaticallyCloseIncidents
   value:
-    simple: "False"
+    simple: "True"
   required: false
-  description: Whether to automatically close the incidents that make up the campaign. Can be True or False.
+  description: |-
+    Whether to automatically close the current incident when it's added to a campaign. Can be True or False.
+    Note: this will only close the current incident. When a campaign is detected, the first few incident/s will remain open, because they do not see themselves as part of the campaign (only newer incidents find the older ones). That number of incidents is the number that you specified in the "minIncidentsForCampaign" playbook input.
```

Comment:
```suggestion
    Note: This will only close the current incident. When a campaign is detected, the first few incident/s will remain open, because they do not see themselves as part of the campaign (only newer incidents find the older ones). That number of incidents is the number that you specified in the "minIncidentsForCampaign" playbook input.
```